docs/core/deploying/creating-nuget-packages.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Create a NuGet Package with .NET Core command-line interface (CLI) tools
+title: Create a NuGet Package with .NET Core command-line tools
```

Comment:
We can actually just say .NET Core CLI 
https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=44109

```suggestion
title: Create a NuGet Package with .NET Core CLI
```